features/docs/defining_steps/ambiguous_steps.feature

```diff
@@ -0,0 +1,63 @@
+Feature: Ambiguous Steps
+ 
```

Comment:
It would be good to add some documentation here about this feature. Something like:

```
When Cucumber searches for a step definition for a step, it might find multiple step
definitions that could match. In that case, it will give you an error that the step 
definitions are ambiguous.

You can also use a `--guess` mode, where it uses magic powers to try and figure 
out which of those two step definitions is most likely to be the one you meant it 
to use. Use it with caution!
```